Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -42,9 +213,78 @@ def test_module(client: PolicyComputeEngine) -> str:
     raise ValueError("Failed to establish connection with provided credentials.")
 
 
+def illumio_traffic_analysis_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """Retrieve the traffic for a particular port and protocol.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns: CommandResult object
+    """
+    port = arg_to_number(args.get('port'))
+    protocol = args.get('protocol', '').lower()
+    start_time = arg_to_datetime(args.get('start_time')).isoformat()  # type: ignore
+    end_time = arg_to_datetime(args.get('end_time')).isoformat()  # type: ignore
+    policy_decisions = argToList(args.get('policy_decisions'))
+    validate_required_parameters(port=port, protocol=protocol, start_time=start_time, end_time=end_time,
```

Comment:
The virtual service create function is validating its required params as part of its dedicated validation function - we should be consistent about that.

Reply:
We're validating required params separately in the command function, will follow this throughout the development